adminpage/sport/models/semester.py

```diff
@@ -1,8 +1,13 @@
+from datetime import datetime
+
 from django.db import models
 
 
 class Semester(models.Model):
     name = models.CharField(max_length=50, null=False, unique=True)
+    start = models.DateTimeField(null=False, default=datetime.now)
```

Comment:
Don't use default datetime tz-unaware objects in django with timezone setting enabled ([documentation](https://docs.djangoproject.com/en/3.0/topics/i18n/timezones/ )) .
Use
```
from django.utils import timezone

now = timezone.now()
```